src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,206 @@
+$base-color: #000066;
+$white-bg: #fff;
+$link-color: #939393;
+
+.cardOuter {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  padding: 20px;
+  margin: 200px 30px;
+  justify-content: center;
+  background-color: $white-bg;
+  color: $base-color;
+  position: relative;
+  border-radius: 15px;
+}
+
+.cardMain {
+  flex-basis: 70%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.cardImage {
+  flex-basis: 30%;
+  padding: 0 2em;
+
+  img {
+    height: 150px;
+    border-radius: 10px;
+  }
+}
+
+.cardInfo {
+  flex-basis: 70%;
+}
+
+.cardContact {
+  flex-basis: 30%;
+  display: flex;
+  flex-direction: column;
+  padding-top: 1em;
+}
+
+.cardLinks {
+  flex-basis: 100%;
+}
+
+.contactInfo {
+  flex-basis: 100%;
+}
+
+.textIcon {
+  padding-left: 20px;
+  position: relative;
+  vertical-align: bottom;
+  
+  span {
+    display: inline-block;
+    width: 20px;
+    left: 0;
+    position: absolute;
+    top: 0;
+    height: 100%;
+    
+    i {
+      font-size: 1.2em;
+      font-weight: 500;
+      left: 0;
+      position: absolute;
+      top: 0;
+      }
+  }
+}
+
+.card-links {
+  text-align: left;
+  margin: 1em 0;
+  i {
+    color: $base-color;
+    font-size: 2em;
+    margin-right: 1em;
+  }
+}
+
+.trackPerson {
+  position: absolute;
```

Comment:
same here. please read - https://teamtreehouse.com/community/when-do-i-use-absolute-positioning-and-when-do-i-use-relative-positioning